applications/api.py

```diff
@@ -91,20 +91,21 @@ def get_required_submission_type(application):
     )
 
 
-def process_upload_resume(resume_file, bootcamp_application):
+def process_upload_resume(resume_file, linkedin_url, bootcamp_application):
```

Comment:
Looking at this again, I'm leaning toward getting rid of this API function and moving this logic into the view itself. It's just doing a single bit of validation (which I'm pretty sure is redundant), calling a model method, and saving. Until we have a another use case for running this same logic, I don't think we get much out of splitting this into a helper function. This would simplify tests for us as well